### ISSUE TITLE
Dialoghost Partially Displayed in WPF

### ISSUE THREAD

**Issue by dragonballz96** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2034]
I just discovered that MD popup dialogs in my WPF app (C#, .Net 4.8) will not displayed properly anymore. I have tried the sample app and it has the same issue as shown below: 

![md dialog issue](https://user-images.githubusercontent.com/25214516/90983204-01253d80-e59f-11ea-9dc1-67cc35813e6e.PNG)

I just wonder if this issue was caused by Windows or .Net framework updates. Anyone facing similar situation? 

Please advise. Thanks.

**Comment by Keboo** [comment]
I am not currently able to reproduce this with the latest code or the [3.1.3 release](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/releases/tag/v3.1.3). Can you provide more information on how to reproduce this?

**Comment by dragonballz96** [comment]
I have attached the clip to demonstrate the issue in the sample app.

[IMG_0959.zip](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/files/5115798/IMG_0959.zip)

I am using VS2019 Pro, 16.7.2 and running in Win 10 Pro environment.

I am suspecting the recent Windows Updates indicated below might be the root cause. 

![image](https://user-images.githubusercontent.com/25214516/91005040-67927600-e608-11ea-8994-37a55750f457.png)

I am going to remove the updates and see if it works.

**Comment by dragonballz96** [comment]
> I have attached the clip to demonstrate the issue in the sample app.
> 
> [IMG_0959.zip](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/files/5115798/IMG_0959.zip)
> 
> I am using VS2019 Pro, 16.7.2, MD 3.1.3 and running in Win 10 Pro environment.
> 
> I am suspecting the recent Windows Updates indicated below might be the root cause.
> 
> ![image](https://user-images.githubusercontent.com/25214516/91005040-67927600-e608-11ea-8994-37a55750f457.png)
> 
> I am going to remove the updates and see if it works.

I ha

**Comment by dragonballz96** [comment]
I have tried removing the Windows updates but the dialoghost was still not displaying properly. Any advice?

**Comment by jespersh** [comment]
Could you try software rendering? See https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/1097#issuecomment-434181412

**Comment by dragonballz96** [comment]
> Could you try software rendering? See [#1097 (comment)](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/1097#issuecomment-434181412)

Hi. Indeed it was working after adding the following code:
RenderOptions.ProcessRenderMode = System.Windows.Interop.RenderMode.SoftwareOnly;

I had identified the issue was caused by my recent upgrade to Parallels Desktop v16. Everything was working fine in previous version (v15). I have already informed them to look into the issue. Thanks for your advice.
